Lombiq.Hosting.Tenants.IdleTenantManagement.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -9,18 +9,22 @@ namespace Lombiq.Hosting.Tenants.IdleTenantManagement.Tests.UI.Extensions;
 public static class TestCaseUITestContextExtensions
 {
     public static async Task TestIdleTenantManagerBehaviorAsync(
-        this UITestContext context)
+        this UITestContext context,
+        string recipeId = null)
     {
         // Setting up new tenant to test the feature
         await context.CreateAndSwitchToTenantManuallyAsync(IdleTenantName, IdleTenantPrefix, string.Empty);
 
+        // This is needed for testing in NuGet as the RecipeId is different.
+        var setupRecipeId = recipeId ?? IdleTenantRecipe;
```

Comment:
I know this is not new, but please also rename it to `DefaultIdleTenantSetupRecipeId`.